libraries/aws_s3_bucket.rb

```diff
@@ -68,6 +68,17 @@ def has_access_logging_enabled?
     end
   end
 
+  def prevent_public_access?
+    return false unless exists?
+    catch_aws_errors do
+      public_access_config = @aws.storage_client.get_public_access_block(bucket: @bucket_name).public_access_block_configuration
+      @prevent_public_access ||= public_access_config.block_public_acls == true &&
+                                 public_access_config.ignore_public_acls == true &&
+                                 public_access_config.block_public_policy == true &&
+                                 public_access_config.restrict_public_buckets == true
+    end
```

Comment:
```suggestion
    @prevent_public_access ||= catch_aws_errors do
      public_access_config = @aws.storage_client.get_public_access_block(bucket: @bucket_name).public_access_block_configuration
       public_access_config.block_public_acls == true &&
                                 public_access_config.ignore_public_acls == true &&
                                 public_access_config.block_public_policy == true &&
                                 public_access_config.restrict_public_buckets == true
    end
```

Reply:
@sathish-progress, rubocop is not able to define the function in the suggested manner: 
![image](https://user-images.githubusercontent.com/42916862/164816308-2a618269-38ec-4eda-8f65-685e6138b9fe.png)

I am getting this error: ```Use 2 (not 0) spaces for indenting an expression spanning multiple lines. (convention:Layout/MultilineOperationIndentation)```